oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/LoginEndpoint.java

```diff
@@ -50,4 +57,12 @@ public void setPassword(String password) {
     this.password = password;
   }
 
+  public String getErrormessage() {
```

Comment:
These are not used. Usually getters and setters are to be avoided. 
If you want more on the matter : https://dev.to/scottshipp/avoid-getters-and-setters-whenever-possible-c8m